pages/common/img2pdf.md

```diff
@@ -0,0 +1,32 @@
+# img2pdf
+
+> Command-line lossless converter of raster images to PDF.
+> More information: <https://gitlab.mister-muffin.de/josch/img2pdf>.
+
+- Convert images to PDF:
+
+`img2pdf {{path/to/image1.jpg}} {{path/to/image2.jpg}} --output {{path/to/file.pdf}}`
+
+- Convert all PNGs in the current directory to a single pdf:
+
+`img2pdf {{*.png}} --output {{path/to/file.pdf}}`
+
+- Convert only the first page of multi-page images to pdf:
+
+`img2pdf {{path/to/file.gif}} --first-frame-only --output {{path/to/file.pdf}}`
+
+- Auto orient to A4 page size in landscape mode and a border of 2cm horizontally and 5.1cm vertically:
+
+`img2pdf {{path/to/file.jpg}} --auto-orient --pagesize {{A4^T}} --border {{2cm:5.1cm}} --output {{path/to/file.pdf}}`
+
+- Shrink only larger images to a 10cm by 15cm rectangle inside a 30x20cm page:
+
+`img2pdf {{path/to/file.tiff}} --pagesize {{30cmx20cm}} --imgsize {{10cmx15cm}} --fit {{shrink}} --output {{path/to/file.pdf}}`
+
+- Convert an image to PDF including specific metadata:
+
+`img2pdf {{path/to/file.png}} --title {{title}} --author {{author}} --creationdate {{1970-01-31}} --keywords {{keywords}} --subject {{subject}} --output {{path/to/file.pdf}}`
+
+- Force RGB as colorspace:
```

Comment:
To me this description is a bit unclear. If I have a YUV file, does it convert YUV to RGB? Does it read my YUV image as RGB?

For the first one, you could use something like `Convert an image to PDF with the RGB colorspace`

Reply:
After a few tests, I couldn't figure out what this did in a practice. If I used the same colorspace as the input image, then no change. In contrast, other options would completely mess with the image. I assume it treats the file as a specific colorspace despite what the metadata may say. I could remove this line, doesn't seem necessary for a TLDR.

Reply:
Sounds good